pkg/apis/servicecatalog/testing/fuzzer.go

```diff
@@ -65,6 +66,17 @@ func createParameter(c fuzz.Continue) (*runtime.RawExtension, error) {
 	return &runtime.RawExtension{Raw: b}, nil
 }
 
+func createParameters(c fuzz.Continue) *servicecatalog.Parameters {
```

Comment:
Since this seems to only be used for instances, and you renamed the other one createBindingParameters, how about calling this one createInstanceParameters just for consistency/clarity?  minor nit.